Dockerfile

```diff
@@ -1,34 +1,30 @@
 # Based from https://github.com/paritytech/substrate/blob/master/.maintain/Dockerfile
 
-FROM phusion/baseimage:bionic-1.0.0 as builder
+FROM phusion/baseimage:jammy-1.0.1 as builder
 LABEL maintainer="hi@zeitgeit.pm"
 LABEL description="This is the build stage for the Zeitgeist node. Here is created the binary."
 
 ENV DEBIAN_FRONTEND=noninteractive
 
-ARG PROFILE=release
+ARG PROFILE=production
 ARG FEATURES=default
 WORKDIR /zeitgeist
 
 COPY . /zeitgeist
 
 RUN apt-get update && \
-    apt-get dist-upgrade -y -o Dpkg::Options::="--force-confold" && \
-    apt-get install -y cmake pkg-config libssl-dev git clang libclang-dev
```

Comment:
`cmake` and `pkg-config` were removed.

Reply:
We don't need it in the first stage dockerfile which builds the Zeitgeist binary. I have successfully built the docker image resulting from the dockerfile.